libs/execution_context/ebpf_maps.c

```diff
@@ -104,7 +104,7 @@ typedef struct _ebpf_core_lpm_map
     ebpf_core_map_t core_map;
     uint32_t max_prefix;
     // Bitmap of prefix lengths inserted into the map.
-    uint8_t data[1];
```

Comment:
Two questions: 
1. if this array contains 1 element, does it need to be an array?
2. If uint8_t was enough, might there be any benefit to changing this to uint32_t and using the regular (not 64-bit) interlocked operation?

Reply:
It's not just one element, it's a variable number of elements, but at least one.
And uint8_t was not enough.